Automattic/wp-calypso
# Free Flow: Remove intro page

#### Testing Estimation

Testing Time: Short
Reviewing Time: Short

#### Proposed Changes

* Remove Free flow intro page
* Refactor the login redirect logic -- On production, you are able to access the other screens in the free flow (`freeSetup` & `designSetup`) while you're logged out. Since I was moving this logic as I removed the intro screen related code, I figured I would also fix the logged/logged in redirection for all the free flow screens.

#### Testing Instructions



* Checkout branch
* Confirm `http://calypso.localhost:3000/setup/free/` redirects to `http://calypso.localhost:3000/setup/free/freeSetup`.
* Log out of Wordpress.com and navigate to `calypso.localhost:3000/setup/free`. Login and confirm you are redirected to the free flow setup screen.
* Log out of Wordpress.com and navigate to  `http://calypso.localhost:3000/setup/free/designSetup?siteSlug={siteSlug}`. Login and confirm you are redirected to the free flow design setup screen.
* Log out of Wordpress.com and navigate to  `calypso.localhost:3000/setup/free/launchpad?siteSlug={siteSlug}`. Login and confirm you are redirected to the free flow launchpad screen.

#### Pre-merge Checklist





Related to #72427
